src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionQRScannerViewController.swift

```diff
@@ -46,13 +46,29 @@ protocol QRScannerViewController: class {
 	func present(_: UIViewController, animated: Bool, completion: (() -> Void)?)
 }
 
-final class ExposureSubmissionQRScannerNavigationController: UINavigationController {
-	var exposureSubmissionService: ExposureSubmissionService?
+final class ExposureSubmissionQRScannerNavigationController: UINavigationController, ExposureSubmissionCoordinatorViewController {
 
+	// MARK: - Attributes.
```

Comment:
please leave break inbetween MARK and code. you did it for the rest of the change as well